### PR TITLE
[Fix] Add loader to swapr balance

### LINKED ISSUE #604: Add loader for SWPR balance in the Header
Currently there is no loading indicator for swapr balabce like for native currency balance
https://user-images.githubusercontent.com/33226956/151814139-45704020-a048-4026-befa-da46dd715d51.mov
@zamli Any suggestion in terms of how it should look like or should we make it same as for native?

### PR DESCRIPTION
**Pull request by berteotti**
- Closes #604

### CODE REVIEW

**Review by Mi-Lan — APPROVED** [review]
Looks good now!

File: src/components/Header/index.tsx [hunk 1]
```diff
@@ -280,10 +280,10 @@ function Header() {
           <Amount zero={!!userNativeCurrencyBalance?.equalTo('0')}>
             {!account ? (
               '0.000'
-            ) : userNativeCurrencyBalance ? (
-              userNativeCurrencyBalance?.toFixed(3)
-            ) : (
+            ) : !userNativeCurrencyBalance ? (
               <Skeleton width="40px" />
+            ) : (
+              userNativeCurrencyBalance?.toFixed(3)
```
Comment (adamazad) [inline src/components/Header/index.tsx:286]: Optional chaining is not required here. if `userNativeCurrencyBalance` is undefined, the `Skeleton` component is shown.

**Review by adamazad — APPROVED** [review]

File: src/components/Header/index.tsx (hunk 1 above)
Comment (berteotti) [reply src/components/Header/index.tsx]: Fixed!

**Review by adamazad — APPROVED** [review]
Great work!

**Pull request merged**